documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -176,8 +169,8 @@ You can also use ``std::vector`` with a USM allocator, as shown in the following
     // Recommended to use USM pointers:
     std::sort(policy, vec.data(), vec.data() + vec.size());
 
-    // Iterators for USM allocators might require extra copying - not recommended method
-    // std::sort(policy, vec.begin(), vec.end());
+    // Iterators for USM allocators might require extra copying - not a recommended method
+    std::sort(policy, vec.begin(), vec.end());
```

Comment:
This was intentionally left commented out to help indicate it is not a recommended way of calling sort.  
I don't think it makes a lot of sense to have this uncommented here.  We would be sorting an already sorted vector.  
Perhaps instead of removing the comment, we would be better formatting this in a different way if it seems like the comment is unintentional. Maybe something like this?
```
    /* Iterators for USM allocators might require extra copying - not recommended method
        std::sort(policy, vec.begin(), vec.end()); 
    */
```